sections/faqs/nextjs-13-issues.mdx

```diff
@@ -0,0 +1,73 @@
+## Why do I see a flash of unstyled content (FOUC) in my Next.js 13 app?
+
+When using Next.js 13 with styled-components, you may experience a flash of unstyled content (FOUC) due to "delay bugs" in client-side rendering. This happens because styled-components' styles are not immediately applied when the page is rendered, causing unstyled elements to appear for a split second.
+
+To resolve this issue, follow these steps:
+
+1. Add the following to your next.config.js file.
+
+```js
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
```

Comment:
Strict mode required?

Reply:
That's there because I copy-pasted the solution from #927. I can delete it if unnecessary.

Reply:
Also, let me know if I should take out strict mode